tests/conftest.py

```diff
@@ -525,3 +527,40 @@ def related_images_refs(admin_client: DynamicClient) -> set[str]:
     related_images = get_csv_related_images(admin_client=admin_client)
     related_images_refs = {img["image"] for img in related_images}
     return related_images_refs
+
+
+@pytest.fixture(scope="session")
+def os_path_environment() -> str:
+    return os.environ["PATH"]
+
+
+@pytest.fixture(scope="session")
+def bin_directory(tmpdir_factory: TempdirFactory) -> LocalPath:
+    return tmpdir_factory.mktemp(basename="bin")
+
+
+@pytest.fixture(scope="session")
+def bin_directory_to_os_path(os_path_environment: str, bin_directory: LocalPath, oc_binary_path: str) -> None:
+    LOGGER.info(f"OC binary path: {oc_binary_path}")
+    LOGGER.info(f"Adding {bin_directory} to $PATH")
+    os.environ["PATH"] = f"{bin_directory}:{os_path_environment}"
+
+
+@pytest.fixture(scope="session")
+def oc_binary_path(bin_directory: LocalPath) -> str:
+    installed_oc_binary_path = os.getenv("OC_BINARY_PATH")
+    if installed_oc_binary_path:
+        LOGGER.warning(f"Using previously installed: {installed_oc_binary_path}")
+        return installed_oc_binary_path
+
+    return download_oc_console_cli(tmpdir=bin_directory)
+
+
+@pytest.fixture(scope="session", autouse=True)
+@pytest.mark.early(order=0)
+def autouse_fixtures(
+    bin_directory_to_os_path: None,
+    cluster_sanity_scope_session: None,
+) -> None:
+    """Fixture to control the order of execution of some of the fixtures"""
+    return
```

Comment:
what is this?

Reply:
It seems like this doesn't do anything, are you using it just to trigger `bin_directory_to_os_path` and `cluster_sanity_scope_session`?
If so, why not call those at session start or add them as autouse?